src/MOCK_DB_DATA/library-items/color/color-system.ts

```diff
@@ -30,56 +30,56 @@ function getExampleHtml(): string {
 
```

Comment:
I think we should not expose core variables (only aliases, but that's something we can do later)